lib/Doctrine/DBAL/Schema/Table.php

```diff
@@ -328,6 +328,8 @@ public function addColumn($columnName, $typeName, array $options=array())
     /**
      * Renames a Column.
      *
+     * @deprecated
```

Comment:
There should be an explanation of the deprecation pointing to what to do instead (just like we should never throw an exception with an empty message for instance)